rust/metaplex/program/src/instruction.rs

```diff
@@ -123,6 +123,7 @@ pub enum MetaplexInstruction {
     ///   18. `[optional/writable]` Master edition (if Printing type of WinningConfig)
     ///   19. `[optional/writable]` Reservation list PDA ['metadata', program id, master edition key, 'reservation', auction manager key]
     ///        relative to token metadata program (if Printing type of WinningConfig)
+    ///   20. `[]` Auction extended
```

Comment:
I normally leave comment like PDA of ['auction', program id, resource, 'extended'] relative to auction contract next to this so people know how to derive it(I think that's the correct seed).